src/components/suggest/view.js

```diff
@@ -14,7 +14,7 @@ export default class SuggestView extends View {
       this.el.querySelectorAll('.input-symptom').forEach((item) => {
         // we do not mark any symptoms that comes from suggest as absent
         if (item.checked) {
-          group[item.id] = {reported: true, related: true};
+          group[item.id] = {reported: true, related: true, source: 'suggest'};
```

Comment:
I think that `source: suggest` should replace related: true. Isn't that true?

Reply:
Also, please check if `reported` is being used anywhere 🤔

Reply:
### Related:

Good point, suggest replaces related, I removed in one place and forgot to remove here, now it is gone. 🚀 

Btw. (this did not affect the application because the flag was never used, but I agree with your point 100%, if it's not used let's remove it)

### Reported:

I double-checked and this flag is part of the logic determining `choice_id` if it is `true` marks `choice_id` as `present` and if is `false` marks `choice_id` as absent. From an architectural point of view maybe it's not needed because we could assign the values directly, but it works and I don't see a point in changing it 😄